src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random RAND = new Random();
+    private static final ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
```suggestion
    private final Random random = new Random();
    private final ColorSupplier colorSupplier = new ColorSupplier();
```
let's not make them static